modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1218,6 +1224,70 @@ class Edit_User extends \NDB_Form
         return $permission;
     }
 
+    /**
+     * This function returns a boolean representing whether or not
+     * to display examiner fields to a user that is editing an account.
+     * Examiner fields SHOULD be displayed in the following cases:
+     *      (1) Editor has 'examiner_multisite' permission
+     *          (which allows to add / certify examiners across all sites).
+     *      (2) Editor has 'examiner_view' permission
+     *          (which allows them to add / certify examiner at their own site)
+     *              AND  either:
+     *                  i) The user is from same site as editor,
+     *                      OR
+     *                  ii) The editor is creating a new user.
+     *
+     * @param \User $editor User editing the form
+     *
+     * @return bool  - whether or not to display examiner fields
+     */
+    function showExaminerFields($editor)
+    {
+        $siteMatch = false;
+        if (!$this->isCreatingNewUser()) {
+            $user        = \User::factory($this->identifier);
+            $userSites   = $user->getCenterIDs();
+            $editorSites = $editor->getCenterIDs();
+            foreach ($userSites as $site) {
+                if (in_array($site, $editorSites)) {
+                    $siteMatch = true;
```

Comment:
Hi @jesscall, shouldn't you add a `break;` after $siteMatch = true; and so you break out of the foreach for the performance benefit of not having to continue looping when you found the match?

Reply:
good point! I've made the suggested change